tests/src/Simple/PInvoke/PInvoke.cs

```diff
@@ -67,8 +67,17 @@ internal class Program
         private static extern bool VerifySizeParamIndex(
                 [MarshalAs(UnmanagedType.LPArray, SizeParamIndex = 1)] out byte[] arrByte, out byte arrSize);
 
-        [DllImport("*", CallingConvention = CallingConvention.StdCall, CharSet = CharSet.Unicode)]
-        private static extern int VerifyStringBuilder(StringBuilder sb);
+        [DllImport("*", CallingConvention = CallingConvention.StdCall, CharSet = CharSet.Unicode, EntryPoint = "VerifyUnicodeStringBuilder")]
+        private static extern int VerifyUnicodeStringBuilder(StringBuilder sb);
```

Comment:
We don't support Ansi array case yet. I will send out another PR with that and add test cases for it.